src/EnergyPlus/MixedAir.hh

```diff
@@ -125,17 +125,22 @@ namespace MixedAir {
     };
 
     // OA Controller Limiting Factor (used for integer output variable values for OAControllerProps::OALimitingFactor
-    // can't change these to enum class since these are used in SetupOutputVariable()
-    constexpr int limitFactorNone = 0;        // No limit other than fixed OA amount
-    constexpr int limitFactorLimits = 1;      // Limits and scheduled limits
-    constexpr int limitFactorEconomizer = 2;  // Economizer operation
-    constexpr int limitFactorExhaust = 3;     // Exhaust flow
-    constexpr int limitFactorMixedAir = 4;    // Mixed air flow rate
-    constexpr int limitFactorHighHum = 5;     // High humidity economizer control
-    constexpr int limitFactorDCV = 6;         // Demand-controlled ventilation
-    constexpr int limitFactorNightVent = 7;   // Night ventilation
-    constexpr int limitFactorDemandLimit = 8; // Demand-limiting
-    constexpr int limitFactorEMS = 9;         // EMS override
+    // stored as int since it is used in SetupOutputVariable()
+    enum class OALimitFactor
+    {
+        Invalid = -1,
+        None = 0,        // No limit other than fixed OA amount
```

Comment:
Don't need the numbers.

Reply:
> Don't need the numbers.

Yes, but these get used as output variable values which are documented in the I/O Ref, so the numbers are here to emphasize that these shouldn't be rearranged or have anything inserted in the middle of the list.